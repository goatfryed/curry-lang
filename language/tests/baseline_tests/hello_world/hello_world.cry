printf("Hello World! Greetings from curry-lang")
