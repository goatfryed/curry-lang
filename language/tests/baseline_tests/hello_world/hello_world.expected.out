"Hello World! Greetings from curry-lang"